functional/cypress-tests/cypress/integration/dashboards.js

```diff
@@ -1,175 +1,190 @@
+            .click()
+        cy.contains('[data-cy="board-card"]', dashboardName)
+            .should('not.exist')
+    })
+})
+
+describe('Dashboard Frontend Validation', () => {
+    beforeEach(() => {
+        cy.visit('/')
+        cy.login()
+    })
+
+    it(' For empty dashboard name is displayed and submit is impossible', () => {
+        cy.addDashboard(' ', '4', '10', true)
+        cy.get('[data-cy="board-form-title-input-error"]')
+            .should('is.visible')
+    })
+
+    dashboardNames.forEach(value => {
+        it(` For too long dashboard name is displayed and submit is impossible. Length: ${value.length}`, () => {
+            cy.addDashboard(value, '8', '10', true)
+            if (value.length > 25) {
+                cy.contains(
+                    '[data-cy="board-form-title-input-error"]',
+                    'Title length'
+                )
+                    .should('is.visible')
+            } else {
+                cy.get('[data-cy="board-form-title-input-error"]')
+                    .should(
+                        'not.visible'
+                    )
+            }
+        })
+    })
+
+    columnEdgeValues.forEach(value => {
+        it(` For Columns input is displayed and submit is impossible for incorrect values. Edge value : ${value}`, () => {
+            cy.addDashboard(dashboardNameGen(), value, '10', true)
+            if (value == '3' || value == '21') {
+                cy.get('[data-cy="board-form-columns-input-error"]')
+                    .should(
+                        'is.visible'
+                    )
+            } else {
+                cy.get('[data-cy="board-form-columns-input-error"]')
+                    .should(
+                        'not.visible'
+                    )
+            }
+        })
+    })
+
+    switchIntervalEdgeValues.forEach(value => {
+        it(` For Switch Interval input is displayed and submit is impossible for incorrect values. Edge value: ${value}`, () => {
+            cy.addDashboard(dashboardNameGen(), '8', value, true)
+            if (value == '2') {
+                cy.get('[data-cy="board-form-switch-interval-input"]')
+                    .should(
+                        'is.visible'
+                    )
+            } else {
+                cy.get('[data-cy="board-form-switch-interval-input"]')
+                    .should(
+                        'not.visible'
+                    )
+            }
+        })
+    })
+})
+
+describe('Dashboard switcher', () => {
+    beforeEach(() => {
+        cy.visit('/')
+        cy.login()
+    })
+
+    it('Manual switching works', () => {
+        const dashboardName = dashboardNameGen()
+        let prevDashboardName, nextDashboardName
+
+        cy.addDashboard(dashboardName, '8', '3')
+        cy.chooseDashboard(dashboardName)
+        cy.contains('[data-cy="navbar-title-header"]', `${dashboardName}`)
+            .should('is.visible')
```

Comment:
Here `should()` and `'is.visible'` is in one line.

Reply:
I've added new rule for this. Only 3 or more arguments will be required to be stated in multiple lines, otherwise there will be a warning.